paket.dependencies

```diff
@@ -0,0 +1,13 @@
+framework: net48
+content: none
+
+source http://artrepo.daptiv.com/artifactory/api/nuget/nuget-virtual
+
+nuget Common.Logging 3.0.0
+nuget Common.Logging.Core 3.0.0
+nuget FluentAssertions 3.2.1
+nuget log4net 2.0.3
+nuget Moq 4.2.1409.1722
+nuget NuGet.CommandLine
```

Comment:
remove

Reply:
These are needed for example and test projects